### DISCUSSION TITLE
Is it possible to use Next.js with local file protocol

### DISCUSSION THREAD

**Discussion by tjperry07** [vercel/next.js#49461] [Help]
### Summary

I have a use case where my end users can't access the internet or use a local server. Is it possible to use Next.js with local file protocol. AKA open the folder and now you can browse the build?

I did try it very quickly as a test, but of course none of the links, css, or images works.

**Comment by skrd-18** [comment]
Hello, I am interested to know regarding what OP has mentioned albeit 1 year ago XD. I have a similar use case whereby I would require the final web application to be served over the file:// protocol. 

I have a couple of questions. 

1. How do the bundling and optimization differences between Next.js and Vite affect the usability and performance in a local, offline setup?
2. Are there significant differences in the final build size and load time between using Next.js and Vite for this project, and would one option lead to a noticeably faster offline experience?

apologies if my questions seem to simple, I am a beginner in web development. I am trying to learn as much as possible. :)

Thank you !
